website/server/server.js

```diff
@@ -14,30 +16,57 @@ app.use(cors());
 mongoose.connect('mongodb://localhost:27017/local');
 
 const Clothes = mongoose.model('Clothes');
+const Category = mongoose.model('Category');
 
 app.use(express.static(__dirname + '/public'));
 
 app.use('/api/clothes', clothesRouter);
 
 app.get('/createData', (req, res) => {
     console.log('run');
-    const clothes = new Clothes({
-        _id: mongoose.Types.ObjectId(),
-        size: 'XLXS',
-        price: 300000,
-        name: 'Collete Stretch Linen Minidress',
-        brand: 'imk',
-        img: '/img/girl_child.jpg'
+    Clothes.find()
```

Comment:
Why don't you create a route for dumping data? then you can update others schema as well, likes: /dump-data/product, or /dump-data/category.